exporters/otlp/src/main/java/io/opentelemetry/exporter/otlp/OtlpGrpcSpanExporter.java

```diff
@@ -105,6 +99,12 @@
    *     or to a negative value, the exporter will wait indefinitely.
    */
   private OtlpGrpcSpanExporter(ManagedChannel channel, long deadlineMs) {
+    Meter meter = GlobalMetricsProvider.getMeter("io.opentelemetry.exporters.otlp");
+    this.spansSeen =
+        meter.longCounterBuilder("spansSeenByExporter").build().bind(EXPORTER_NAME_LABELS);
+    LongCounter spansExportedCounter = meter.longCounterBuilder("spansExportedByExporter").build();
+    this.spansExportedSuccess = spansExportedCounter.bind(EXPORT_SUCCESS_LABELS);
+    this.spansExportedFailure = spansExportedCounter.bind(EXPORT_FAILURE_LABELS);
```

Comment:
I thought we had already done this. Good catch!

Reply:
I need to unbind them in shutdown.

Reply:
Done.